tensorflow/lite/tools/make/Makefile

```diff
@@ -128,6 +132,9 @@ BUILD_WITH_NNAPI=true
 ifeq ($(BUILD_TYPE),micro)
```

Comment:
Let's also make the windows change for mmap_allocation.cc vs mmap_allocation_disabled.cc above, and then let's undo the changes to mmap_allocation.cc and nnapi_delegate.cc